TwitchClient/Game.swift

```diff
@@ -1,35 +0,0 @@
-//
-//  Game.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Game: NSObject {
```

Comment:
I will not repeat the comments previously mentioned. This is why I'm recommending topics to read on, if you actually grasp why something is better than another then you'll be able to apply the same logic going forward.